src/Umbraco.Web/HtmlStringUtilities.cs

```diff
@@ -235,10 +245,79 @@ public IHtmlString Truncate(string html, int length, bool addElipsis, bool treat
                     outputms.Position = 0;
                     using (TextReader outputtr = new StreamReader(outputms))
                     {
-                        return new HtmlString(outputtr.ReadToEnd().Replace("  ", " ").Trim());
+                        string result = String.Empty;
+
+                        string firsTrim = outputtr.ReadToEnd().Replace("  ", " ").Trim();
+                        
+                        //Check to see if there is an empty char between the hellip and the output string
+                        //if there is, remove it
+                        result = firsTrim[firsTrim.Length -9] == ' ' ? firsTrim.Remove(firsTrim.Length - 9, 1) : firsTrim;
```

Comment:
@RobertCopilau  I'm just wondering if this can cause an error since this is not checking if the `firsTrim` string has enough length in it to do a `firsTrim[firsTrim.Length -9]` is there a guarantee that `firsTrim` will absolutely be that long here?

Also, `firsTrim` should be `firstTrim` i think?

Reply:
Yes, you are correct, it will throw an error, I will get it done in a few minutes. And yes it was supposed to be firstTrim :).